packages/core/package.json

```diff
@@ -61,16 +67,16 @@
     "style-loader": "^3.3.1",
     "swr": "^1.3.0",
     "tsconfig-paths-webpack-plugin": "^3.5.2",
-    "typescript": "4.7.3"
+    "tsx": "^3.12.7",
```

Comment:
Aren't `tsx`, `prettier` and `@vtex/prettier-config` `devDependencies`?

Reply:
They are in the context of this repository, but we want them to ship with @faststore/core so it can use these packages when used by the starter. I'll double check on the necessity of including tsx, I'm not sure why it's there.